.github/module-filter.yml

```diff
@@ -0,0 +1,81 @@
+common: &common
```

Comment:
Nice, I like this approach - I didn't know you could pass the path-filters action a yml and use features like anchors.